server.js

```diff
@@ -0,0 +1,115 @@
+const express = require('express');
+const favicon = require('express-favicon');
+const path = require('path');
+const port = process.env.PORT || 8080;
+
+// packages import
+const app = express();
+app.use(express.json());
+//app.use(favicon(__dirname + '/build/favicon.png'));
+
+//give static
+//app.use(express.static(__dirname));
+app.use(express.static(path.join(__dirname, 'dist')));
+app.use(express.static(path.join(__dirname, 'static/media')));
+app.use(express.static(path.join(__dirname, 'static/svg')));
+
+//test
+app.get('/ping', function (req, res) {
+    return res.send('pong');
```

Comment:
@IgorChaikin Could you configure your IDE to use `2 spaces` indentation, like it done in `scss` files. You can also configure `prettier` to format your files on every save